app/javascript/shared/helpers/KeyboardHelpers.js

```diff
@@ -106,6 +106,10 @@ export const hasPressedCommandPlusKKey = e => {
   return e.metaKey && e.keyCode === 75;
 };
 
+export const hasPressedCommandZKey = e => {
```

Comment:
Let us not add methods like this. Please use the buildHotKeys methods written below this.

Reply:
@pranavrajs Done.